content/en/docs/how-tos/contributing-openshift-release.md

```diff
@@ -113,9 +114,8 @@ The following changes are considered when triggering rehearsals:
 1. Changes to templates (`ci-operator/templates`)
 1. Changes to cluster profiles (`cluster/test-deploy`)
 
-It is not possible to rerun individual rehearsal jobs. They do not react to any trigger commands. Rerunning rehearsals
-must be done by re-triggering the plugin: `/pj-rehearse`, which then triggers all rehearsals of jobs currently
-affected by the PR, including the rehearsals that passed before.
+Rerunning rehearsals can be done by re-triggering the plugin: `/pj-rehearse`, which then triggers all rehearsals of jobs currently
+affected by the PR, including the rehearsals that passed before. Individual rehearsals can also be re-triggered by utilizing the ['Rehearse Specific Jobs'](#rehearse-specific-jobs) functionality.
```

Comment:
```suggestion
affected by the PR, including the rehearsals that passed before. Individual rehearsals can also be re-triggered by utilizing the [Rehearse Specific Jobs](#rehearse-specific-jobs) functionality.
```
![Screenshot 2022-12-16 at 8 55 48 AM](https://user-images.githubusercontent.com/4013349/208113685-9a4d4fde-f0bc-48f4-a059-a078cf11eb02.png)

I think the colour on the link is good enough.